prow/plugins/config.go

```diff
@@ -975,10 +974,6 @@ func validateConfigUpdater(updater *ConfigUpdater) error {
 					Namespace: namespace,
 					Cluster:   cluster,
 				}
-				if files.Has(file) {
-					return fmt.Errorf("file %s listed more than once in config updater config", file)
```

Comment:
What was this checking before? That we didn't push a file to the same configmap twice? Are we losing some validation?

Reply:
Here is the diff
https://github.com/kubernetes/test-infra/pull/15028/files

Before the error did not show up because we did not do `updater.SetDefaults()` in the `validateConfigUpdater` function. As the new test case showed, when we do `SetDefaults()` it would have 2 namespaces with the same file.

We need `SetDefaults()` here to save the duplicate of the code on set up the cluster, namespaces etc. The  SetDefaults() runs in the code anyway (before my PR) before applying the configs.

Reply:
Originally, it checks if a file show showed up more than once in the config.
I think it is not even possible when it has already become a `map`.